CONTRIBUTING.md

```diff
@@ -177,6 +179,34 @@ To run tests on [SauceLabs](https://saucelabs.com/) locally, you'll need to add
 
 > See more scripts at [SCRIPTS.md](SCRIPTS.md) to learn how to run tests and more.
 
+#### Samples Tests
+
+The samples tests are run by <https://webdriver.io> which spins up two browser instances and has them communicate between each other.
+
+These tests are run with `npm run samples:test`.
+We have found that due to the h.264 codec downloading in Firefox, the best way to run these test is on SauceLabs.
+You can run them on SauceLabs with `SAUCE=true npm run samples:test`.
+
+To run a specific sample test instead of the full suite, modify the [wdio.conf.js](./wdio.conf.js) file's "specs" array like so:
+
+```js
+specs: [
+    './wdio.helpers.d/**/*.js',
+    './packages/node_modules/samples/browser-call-with-screenshare/test/wdio/spec/**/*.js'
+  ],
+```
+
+In this example, you would only run the screenshare test.
+Make sure to leave the helpers line in so that helpers get loaded properly.
+
+##### Local Samples Tests
+
+If you wish to run the samples tests locally, we suggest changing from the Chrome-to-Firefox multiremote setup to Chrome-to-Chrome.
+
+You can do so by modifying the [wdio.conf.js](./wdio.conf.js) file.
+Simply change the `browserSpock`'s `desiredCapabilities` object to the same as `browserMccoy` (the Chrome instance).
```

Comment:
Can we just rename these to Chrome and Firefox now...?
I always get confused which is which

Reply:
^in a later PR of course

Reply:
Good idea! https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-124460